p2p/src/net/default_backend/backend.rs

```diff
@@ -543,95 +549,16 @@ where
         }
     }
 
-    fn handle_message(&mut self, peer: PeerId, message: Message) -> crate::Result<()> {
+    fn handle_message(&mut self, peer: PeerId, message: PeerManagerMessage) -> crate::Result<()> {
         // Do not process remaining messages if the peer has been forcibly disconnected (for example, after being banned).
         // Without this check, the backend might send messages to the sync and peer managers after sending the disconnect notification.
         if !self.peers.contains_key(&peer) {
             log::debug!("ignore received messaged from a disconnected peer");
```

Comment:
Please change this to info

Reply:
Also add the peer info here. This helps in debugging over time without debug mode.